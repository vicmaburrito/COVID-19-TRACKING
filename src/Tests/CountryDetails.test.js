import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configStore';

import CountryDetail from '../Components/Countries/Country_detail';

it('renders correctly', () => {
  const tree = renderer.create(<Provider store={store}><CountryDetail /></Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});
