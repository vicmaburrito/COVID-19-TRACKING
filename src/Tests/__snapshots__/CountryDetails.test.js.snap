// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="d-flex justify-content-center mt-5"
  >
    <span
      className="mt-5 h3"
    >
      Loading...
    </span>
  </div>
</div>
`;
